docs/usage/configuration-options.md

```diff
@@ -568,6 +568,22 @@ In this configuration:
 - The version string is extracted from lines matching `version: <value>`.
 - The `bumpType` is dynamically set to `patch` for if the dependency update is a patch update and `minor` otherwise.
 
+**debugging `bumpVersions`**
+
+Use [`logLevelRemap`](#loglevelremap) to remap, `trace` log level messages to a higher level e.g. `debug`.
+All messages are prefixed with `bumpVersions` or `bumpVersions(<name>)` to help you filter them in the logs.
+
+```json
+{
+  "logLevelRemap": [
+    {
+      "matchMessage": "/bumpVersions/",
+      "newLogLevel": "info"
```

Comment:
```suggestion
      "newLogLevel": "debug"
```
Let's use the same log level as mentioned in the above eg.

Reply:
This is on purpose as these two levels will be the most used target levels.

Reply:
I agree with @RahulGautamSingh that it's best for the config code to match the written description.